umh-core/pkg/sentry/sentry.go

```
@@ -118,12 +116,21 @@ func createSentryEvent(level sentry.Level, err error) *sentry.Event {
 		})
 	}
 
-	// Let Sentry use its default stack trace-based fingerprinting
-	// which is typically more effective for grouping similar errors
+	// Extract the prefix before the first colon if it exists
+	errorMessage := err.Error()
+	fingerprint := "{{ default }}"
```

golangci-lint (1.64.8)

ineffectual assignment to fingerprint

(ineffassign)